PokeAlarm/WebhookStructs.py

```diff
@@ -70,6 +70,7 @@ def pokemon(data):
             'weight': check_for_none(float, data.get('weight'), 'unkn'),
             'gender': get_pokemon_gender(check_for_none(int, data.get('gender'), '?')),
             'size': 'unknown',
+            'form': check_for_none(int, data.get('form'), 0),
```

Comment:
Instead of 0, default to 'unkn' so that we can tell when it is missing vs just unset.